module-2/backend/week_one.md

```diff
@@ -5,30 +5,114 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+The five common verbs: 
+  ** 1. Get ** Get's purpose is to receive information.
+  ** 2. Post ** Post's purpose is to send information.
+  ** 3. Put ** Put's purpose is to update information.
+  ** 4. Delete ** Delete's purpose is to delete information.
+  ** 5. Head ** Head returns just the header, which I imagine is useful for something I haven't encountered yet.
 2. What is Sinatra?
+  * Sinatra is a DSL alternative to Rails that allows me to run a web app. It uses Rack to help communicate with the server.*
 4. What is MVC?
+  *MVC is: Model View Controller. This is the architecture for web apps.*
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  * Using REST makes the code readable to everyone, even if they're new to the project. It makes the code logical to everyone
+  * familiar with the conventions. *
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  *The view templates can access local and instance variables.*
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
     erb :index
   end
   ```
-
+  ***
+  ```ruby
+    get '/horses' do
+      @count = 1
+      erb :index
+    end
+    ```
+    ***
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+    ```ruby
+    get '/horses' do
+      erb :index, locals: {name: "Mr. Ed"}
+    end
+    ```
 9. What's the purpose of ERB?
+  *To have access to ruby to render dynamic views. *
 10. Why do I need a development AND test database?
+  *The test database should be clean so that tests run without data that may have been entered during development. *
 11. What's responsive design?
+  *It's design that works for both desktop and mobile because the screen size is flexible. It responds to the screen size changing. *
 12. What is CRUD and why is it important?
+  * CRUD (Create, Read, Update, Delete) is the foundation for web development. It provides the basic functionality needed for
+  a web app. *
 13. What does HTTP stand for? 
+  *Hyper Text Transfer Protocol*
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+  * 1: <%= test %> 2. <% test %> *
+  * Use 1 when you want to render the Ruby. Use 2 when you want to evaluate it, but not render.*
 15. What's an ORM?
+  * Object Relational Mapping --ActiveRecord is an ORM. *

```

Comment:
Yup! At a higher level it's a tool that we use to map rows in a database to objects that we can use in an OO context.